packages/devtools_app/lib/src/shared/analytics/constants/_deep_links_constants.dart

```diff
@@ -32,6 +32,12 @@ enum AnalyzeFlutterProject {
   /// iOS Universal Links settings are loaded.
   iosUniversalLinkSettingsLoaded,
 
```

Comment:
can you update the comments here to differentiate them from `androidAppLinksSettingsLoaded` and `iosUniversalLinkSettingsLoaded`

Reply:
ah thanks for catching it, copy paste error